src/app/current/current.component.ts

```diff
@@ -8,10 +8,20 @@ import { RequestsService } from '../requests.service';
 })
 
 export class CurrentComponent implements OnInit {
+  //akkor ezt hogy lehet beadni típusnak?
```

Comment:
Don't leave comments (especially like these) in the code! :D
But you should export the type if you want to reach it in another file.